bk-register/ROOTFS/opt/sage/beekeeper/register/register.py

```diff
@@ -46,6 +48,15 @@ def register():
     """
     id = flask.request.args.get("id", type=str)
 
+    # check if BEEKEEPER_DB_API is alive
+    try:
+        bk_api_result = requests.get(f'{BEEKEEPER_DB_API}').content
+    except Exception as e:
+        return f"Error: X Beekeeper DB API ({BEEKEEPER_DB_API}) cannot be reached: {str(e)}"
+
+    if 'SAGE Beekeeper' != bk_api_result.decode("utf-8").strip():
+        return "Error: Y Beekeeper DB API ({BEEKEEPER_DB_API}) cannot be reached: \""+bk_api_result.decode("utf-8")+"\""
```

Comment:
same question (as line 55) about the `return` error code

Reply:
Also I assume you meant to put a number in place of "Y" in the string "Error: Y Beekeeper DB"

Reply:
Also i think you could have used the same `f` format for your string

```
return f"Error: Y Beekeeper DB API ({BEEKEEPER_DB_API}) cannot be reached: \"{bk_api_result.decode("utf-8")}\""
```

Reply:
fixed